From e9d1ec8d413e63ec34ad3cd76cf1c9da7648389b Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Sun, 5 Sep 2021 10:51:46 +0100
Subject: [PATCH 1/1] Disable C++17 check

Unsure why this is failing, when you chroot in and autogen it doesn't fail.
Compiling the check by itself also doesn't fail...

---
 configure.ac | 25 ++-----------------------
 1 file changed, 2 insertions(+), 23 deletions(-)

diff --git a/configure.ac b/configure.ac
index 25da370..4ea951c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -7542,33 +7542,12 @@ elif test "$GCC" = "yes" -o "$COM_IS_CLANG" = TRUE; then
             flag="-Xclang $flag"
         fi
         save_CXXFLAGS=$CXXFLAGS
-        CXXFLAGS="$CXXFLAGS $flag -Werror"
+        CXXFLAGS="$CXXFLAGS $flag -Werror -Wno-deprecated"
         if test "$SYSTEM_LIBCMIS" = TRUE; then
             CXXFLAGS="$CXXFLAGS -DSYSTEM_LIBCMIS $LIBCMIS_CFLAGS"
         fi
         AC_LANG_PUSH([C++])
-        AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
-            #include <algorithm>
-            #include <functional>
-            #include <vector>
-
-            #if defined SYSTEM_LIBCMIS
-            // See ucb/source/ucp/cmis/auth_provider.hxx:
-            #if !defined __clang__
-            #pragma GCC diagnostic push
-            #pragma GCC diagnostic ignored "-Wdeprecated"
-            #pragma GCC diagnostic ignored "-Wunused-but-set-parameter"
-            #endif
-            #include <libcmis/libcmis.hxx>
-            #if !defined __clang__
-            #pragma GCC diagnostic pop
-            #endif
-            #endif
-
-            void f(std::vector<int> & v, std::function<bool(int, int)> fn) {
-                std::sort(v.begin(), v.end(), fn);
-            }
-            ]])],[CXXFLAGS_CXX11=$flag])
+        [CXXFLAGS_CXX11=$flag]
         AC_LANG_POP([C++])
         CXXFLAGS=$save_CXXFLAGS
         if test -n "$CXXFLAGS_CXX11"; then
-- 
2.35.4
